backend/typescript/services/implementations/camperService.ts

```diff
@@ -136,6 +145,17 @@ class CamperService implements ICamperService {
         optionalClauses: newCamper.optionalClauses,
       };
     });
+
+    if (waitlistedCamperId) {
```

Comment:
can we add a try/catch here so if the waitlistedCamperId is invalid we return an error message like `` `Registered campers but was unable to update waitlisted camper status with id: ${waitlistedCamperId}` ``?